src/PrestaShopBundle/Form/Admin/Sell/Product/ProductFormType.php

```diff
@@ -135,6 +136,7 @@ public function buildView(FormView $view, FormInterface $form, array $options)
     {
         $productType = $options['data']['basic']['type'] ?? ProductType::TYPE_STANDARD;
         $formVars = [
+            'productType' => $productType,
```

Comment:
This double data seems redundant, I feel like it means something's not quite right Either we should only have simply accessible form view vars (like `productType` `productId`) that are used to add data attributes to the form in the template

Or the opposite only data attributes that are used in the template to decide how to adapt the display

But the mix of both probably means one is obsolete, the mix of convention (snake case, camel case) also feels weird. It seems Symfony form rather use snake case for their option and view variables like `full_name`, `unique_block_prefix`, ...